01-introduction.Rmd

```diff
@@ -173,7 +173,7 @@ leaflet() |>
              popup = popup)
 ```
 
-```{r interactive, fig.cap="The blue markers indicate where the authors are from. The basemap is a tiled image of the Earth at night provided by NASA. Interact with the online version at r.geocompx.org, for example by zooming in and clicking on the popups.", out.width="100%", fig.scap="Where the authors are from.", echo=FALSE}
+```{r interactive, fig.cap="The blue markers indicate where the authors are from. The basemap is a tiled image of the earth at night provided by NASA. Interact with the online version at r.geocompx.org, for example by zooming in and clicking on the pop-ups.", out.width="100%", fig.scap="Where the authors are from.", echo=FALSE}
```

Comment:
@Robinlovelace Earth->earth is a surprising change for me

Reply:
Me too, it's a planet! Happy to revert.